auth/token.go

```diff
@@ -52,24 +52,33 @@ func RefreshToken(f TokenFunc, filename string) (int64, error) {
 		return 0, err
 	}
 	updateToken(token, time.Unix(expires, 0), filename)
-	log.WithField("expiration", expires).Debug("Received new authentication token")
+	log.WithField("expiration", expires).Info("Received new authentication token")
```

Comment:
If we leave this as Info, it is probably going to show up as output on every CLI command.